modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -89,78 +89,298 @@ This allows your `kubeconfig` file to be consumed from inside the running contai
+I0829 13:25:21.569345       1 node.go:44] Environment information: kernel version 4.18.0-305.10.2.rt7.83.el8_4.x86_64
+I0829 13:25:21.569367       1 main.go:53] Running the oslat command with arguments [--duration 600 --rtprio 1 --cpu-list 4,6,52,54,56,58 --cpu-main-thread 2] <1>
+I0829 13:35:22.632263       1 main.go:59] Succeeded to run the oslat command: oslat V 2.00
+Total runtime: 		600 seconds
 Thread priority: 	SCHED_FIFO:1
-CPU list: 		3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50
+CPU list: 		4,6,52,54,56,58
 CPU for main thread: 	2
 Workload: 		no
 Workload mem: 		0 (KiB)
-Preheat cores: 		48
+Preheat cores: 		6
 
 Pre-heat for 1 seconds...
 Test starts...
 Test completed.
 
-Core: 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50
-CPU Freq: 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2092 2096 2096 2096 2092 2092 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2092 2096 2096 2092 2096 2096 2096 2096 2092 2096 2096 2096 2092 2096 2096 2096 2096 2096 2096 (Mhz)
-...
-Maximum: 3 4 3 3 3 3 3 3 4 3 3 3 3 4 3 3 3 3 3 4 3 3 3 3 3 3 3 3 3 4 3 3 3 3 3 3 3 4 3 3 3 3 3 4 3 3 3 4 (us)
-----
+        Core:	 4 6 52 54 56 58
+    CPU Freq:	 2096 2096 2096 2096 2096 2096 (Mhz)
+    001 (us):	 19390720316 19141129810 20265099129 20280959461 19391991159 19119877333
+    002 (us):	 5304 5249 5777 5947 6829 4971
+    003 (us):	 28 14 434 47 208 21
+    004 (us):	 1388 853 123568 152817 5576 0
+    005 (us):	 207850 223544 103827 91812 227236 231563
+    006 (us):	 60770 122038 277581 323120 122633 122357
+    007 (us):	 280023 223992 63016 25896 214194 218395
+    008 (us):	 40604 25152 24368 4264 24440 25115
+    009 (us):	 6858 3065 5815 810 3286 2116
+    010 (us):	 1947 936 1452 151 474 361
+  ...
+     Minimum:	 1 1 1 1 1 1 (us)
+     Average:	 1.000 1.000 1.000 1.000 1.000 1.000 (us)
+     Maximum:	 37 38 49 28 28 19 (us) <3>
+     Max-Min:	 36 37 48 27 27 18 (us)
+    Duration:	 599.667 599.667 599.667 599.667 599.667 599.667 (sec)
+----
+<1> The list of CPUs running the `oslat` command. Seven CPUs are provided through the `LATENCY_TEST_CPUS` variable. Only six CPUs are displayed in total because one is used to run the `oslat` tool.
+<2> The user is notified about the measured latency and the configured latency.
+<3> The maximum latency values in microseconds that are measured on each CPU.
+
```

Comment:
This block isn't rendering properly for me. The footnotes are showing up as text in a quote block, so something is wrong with the delimiter here.

Reply:
I ran an asciibinder build and seems to work for me, it looks good in the preview as well. Can I ask how did you try to render it?